starlette/convertors.py

```diff
@@ -61,9 +61,20 @@ def to_string(self, value: typing.Any) -> str:
         return ("%0.20f" % value).rstrip("0").rstrip(".")
 
 
-CONVERTOR_TYPES = {
+_DEFAULT_CONVERTOR_TYPES = {
     "str": StringConvertor(),
     "path": PathConvertor(),
     "int": IntegerConvertor(),
     "float": FloatConvertor(),
 }
+
+CONVERTOR_TYPES = dict(_DEFAULT_CONVERTOR_TYPES)
+
+
+def add(key: str, convertor: Convertor) -> None:
+    CONVERTOR_TYPES[key] = convertor
+
+
+def reset() -> None:
```

Comment:
I'm not sure `reset` should be part of the public API - we really only need it for the purposes of our own tests, right? Perhaps we should just have `URL_CONVERTORS` and `register_url_convertor`?

Reply:
Yes it’s only for tests, I guess I can just move this to `test_convertors.py` (and I just realised I spelled that filename wrong).